kindtest/runner_test.go

```diff
@@ -117,6 +117,52 @@ func testRunner() {
 		fmt.Println("====== Pod was actually deleted at " + time.Now().UTC().Format(time.RFC3339))
 	})
 
+	It("should not be visible environment variables on the job", func() {
```

Comment:
Is this explanation correct?
I think this test is checking a lot more things.

Reply:
I wrote a more detailed description of the test.
Is this clear enough? (It should be successful the job that makes sure invisible environment variables.)